aws/sdk/integration-tests/s3/tests/request_id.rs

```diff
@@ -91,7 +91,7 @@ async fn get_request_id_from_unmodeled_error() {
         .expect_err("status 500")
         .into_service_error();
     request.expect_request();
-    assert!(matches!(err, GetObjectError::Unhandled(_)));
+    assert!(err.is_unhandled());
```

Comment:
`is_unhandled` should _maybe_ be deprecated for the same reason

Reply:
I only added it for a unit test. I'll just remove it.